src/@types/styled-component.d.ts

```diff
@@ -14,7 +14,6 @@ declare module "styled-components" {
   export interface DefaultTheme {
     spaces: {
```

Comment:
What about going right for `spaces: Spacing`? Do you think we'll ever need to have two fields for spacing?

Reply:
OK fixed 0d6d25f